website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md

```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
+<TabItem value="core" label="dbt Core">
 
-When an admin changes the dbt Cloud's connection to use the dbt-databricks adapter instead of the dbt-spark adapter, your team will not lose their credentials. This makes migrating from dbt-spark to dbt-databricks straightforward as it only requires deleting the connection and re-adding the cluster/endpoint information. Both the admin and users of the project need not re-enter personal access tokens.
+To migrate your dbt Core projects to the `dbt-databricks` adapter from `dbt-spark`, you:
+1. Install the [dbt-databricks adapter](https://github.com/databricks/dbt-databricks) in your environment
+1. Modify your `target` in your `~/.dbt/profiles.yml` file
 
-#### Procedure
+Anyone who's using your project must also make these changes in their environment.
 
-An admin of the dbt Cloud project running on Databricks should take the following steps to migrate from using the generic Spark adapter to the Databricks-specfic adapter. This should not cause any downtime for production jobs, but we recommend that you schedule the connection change when there is not heavy IDE usage for your team to avoid disruption.
+</TabItem>
+
+<TabItem value="cloud" label="dbt Cloud">
+
+To migrate your dbt Cloud projects to the `dbt-databricks` adapter from `dbt-spark`, you must have admin privileges. The migration shouldn't cause any downtime for production jobs. dbt Labs recommends that you schedule the connection change when usage of the IDE is light to avoid disrupting your team.
 
 1. Select **Account Settings** in the main navigation bar.
-2. On the Projects tab, scroll until you find the project you'd like to migrate to the new dbt-databricks adapter.
+2. On the **Projects** tab, find the project you want to migrate to the dbt-databricks adapter.
 3. Click the hyperlinked Connection for the project.
-4. Click the "Edit" button in the top right corner.
-5. Select Databricks for the warehouse
-6. Select Databricks (dbt-databricks) for the adapter and enter:
-    1. the `hostname`
-    2. the `http_path`
-    3. optionally the catalog name
-7. Click save.
+4. Click **Edit** in the top right corner.
+5. Select **Databricks** for the warehouse
+6. Select **Databricks (dbt-databricks)** for the adapter and enter the:
+    1. `hostname`
+    2. `http_path`
+    3. (optional) catalog name
+7. Click **Save**.
+
+Everyone in your organization who uses dbt Cloud must refresh the IDE before starting work again. It should refresh in less than a minute.
 
-After the above steps have been performed, all users will have to refresh their IDE before being able to start working again. It should complete in less than a minute. 
+#### Your credentials with the migration
```

Comment:
I feel like this section should possibly talk about what you see and what you should do when the authentication doesn't work. I'm a little unclear what my takeaway should be here?

Reply:
Also, should we link to the appropriate docs for how to authenticate if you can't? I think these might be the ones how to link a warehouses?

Reply:
My takeaway when I was going through this content was: You do not need to reenter your credentials with the migration. 

Maybe I should start with that ^ then explain why credentials don't need to be inputted again. Lemme give that a try!